src/nvim/ex_getln.c

```diff
@@ -1781,24 +1781,29 @@ getexmodeline (
         msg_col = startcol;
         msg_clr_eos();
         line_ga.ga_len = 0;
-        continue;
+        goto redraw;
       }
 
+      int num_spaces;
       if (c1 == Ctrl_T) {
         int sw = get_sw_value(curbuf);
 
         p = (char_u *)line_ga.ga_data;
         p[line_ga.ga_len] = NUL;
         indent = get_indent_str(p, 8, FALSE);
-        indent += sw - indent % sw;
+        num_spaces = sw - indent % sw;
+        indent += num_spaces;
```

Comment:
This line is causing a new warning (dead store). Clang says `indent` isn't read any more after this, but on a quick glance I don't see why. Can you confirm is this line can be safely removed?